rocketpy/rocket/rocket.py

```diff
@@ -72,6 +73,9 @@ class Rocket:
         for more information
         regarding the coordinate system.
         Expressed in meters as a function of time.
+    Rocket.z_coordinate_com_to_cdm : Function
+        Function of time expressing the z-coordinate of the center of mass
+        relative to the center of dry mass.
```

Comment:
I have problems with the name `z_coordinate_com_to_cdm`. The name is confusing and we might change the name of the coordinates later

I suggest `com_to_cdm_function`

Reply:
Suggestion accepted. 

[61ebbc7](https://github.com/RocketPy-Team/RocketPy/pull/595/commits/61ebbc754dc41a7436b9ddc5ccc39310fecc4440)